en/development/testing.rst

```diff
@@ -1322,6 +1322,281 @@ comparison files as they are referenced:
     #
     #   modified:   tests/comparisons/example.php
 
+Console Integration Testing
+===========================
+
+To make testing console applications easier, CakePHP comes with a
+``ConsoleIntegrationTestCase`` class that can be used to test console applications
+and assert against their results.
+
+.. versionadded:: 3.5.0
+
+    The ``ConsoleIntegrationTestCase`` was added.
```

Comment:
The version added block already adds that information: https://cl.ly/1K2l1u2P1E0Y

It's a bit redundant to see it twice. There are some inconsistencies in the usage within the docs though, I've seen it with the redundant version (https://cl.ly/32043z00392F) and without (https://cl.ly/1M1L2y190z40) and just plain (https://cl.ly/2G0O0p3M3g06)

If we pick one, I'll go with it :)